### PR TITLE
PD-228: Add pull request template

### PR DESCRIPTION
**Pull request by DesignerDave**
Never again will PR descriptions look like this...

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+## ✍️ Proposed changes
```
Comment (brookescarlett) [inline .github/pull_request_template.md]: 💯

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Describe the big picture of your changes here and communicate why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
```
Comment (hswolff) [inline .github/pull_request_template.md]: What I've seen other projects do is have these notes encased in HTML comment tabs. Such that they show for the submitter, but don't get persisted in the actual PR. `<!-- text -->`.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+
+## ✅ Checklist
+
+_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
+
+- [ ] I have read the [CONTRIBUTING](https://github.com/mongodb/leafygreen-ui/blob/master/CONTRIBUTING.md) doc
+- [ ] Tests pass locally with my changes
+- [ ] I ran `npm run prettier` before committing my changes.
```
Comment (hswolff) [inline .github/pull_request_template.md]: This should go away when we integrate a CI.

Comment (DesignerDave) [reply .github/pull_request_template.md]: Are you saying that the CI will run prettier, or check that prettier has been run? I _think_ you mean the latter.

Either way, yes.

File: .github/pull_request_template.md (hunk 2 above)
Comment (DesignerDave) [reply .github/pull_request_template.md]: Oh does that actually work with GitHub-flavored markdown?

Comment (hswolff) [reply .github/pull_request_template.md]: Seems like it! Check out https://github.com/emotion-js/emotion/blob/master/.github/PULL_REQUEST_TEMPLATE.md

Hit raw to see the comments.

File: .github/pull_request_template.md (hunk 3 above)
Comment (hswolff) [reply .github/pull_request_template.md]: The latter, yes.

**Comment by DesignerDave** [comment]
RFAL @hswolff

File: .github/pull_request_template.md (hunk 2 above)
Comment (DesignerDave) [reply .github/pull_request_template.md]: Done!

**Pull request merged**